src/main/java/com/study/platform/member/domain/MemberController.java

```diff
@@ -0,0 +1,18 @@
+package com.study.platform.member.domain;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.security.oauth2.core.user.OAuth2User;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MemberController {
+
+	@GetMapping("/member")
```

Comment:
REST API는 복수형으로 사용합니다~
```suggestion
	@GetMapping("/members")
```
가 더 좋을거 같아요.